tools/wpt/tests/test_wpt.py

```diff
@@ -13,6 +14,43 @@
 pytestmark = pytest.mark.skipif(os.name == "nt",
                                 reason="Tests currently don't work on Windows for path reasons")
 
+def is_port_8000_in_use():
+    s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    try:
+        s.bind(("127.0.0.1", 8000))
+    except socket.error as e:
+        if e.errno == 98:
```

Comment:
Use the named constant in the `errno` module.

Reply:
Ah, I couldn't find a constant for it anywhere!